scipy/stats/tests/test_mstats_basic.py

```diff
@@ -20,7 +20,7 @@
     assert_array_almost_equal, assert_array_almost_equal_nulp, assert_,
     assert_allclose, assert_array_equal)
 from numpy.testing import suppress_warnings
-
+from .. import mstats_basic
```

Comment:
Rather than assuming the relative location, just import directly.
```
from scipy.stats import mstats_basic
```

Reply:
Sure, changed.